# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../psmisc/no-termcap.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Including the old termcap.h and the term.h results in mixed prototypes ...

  - Rene Rebe <rene@exactcode.de>

diff -u psmisc-21.5/src/pstree.c psmisc-21.5-fixed/src/pstree.c
--- psmisc-21.5/src/pstree.c	2004-04-24 14:44:27.000000000 +0200
+++ psmisc-21.5-fixed/src/pstree.c	2004-12-21 19:04:03.265558192 +0100
@@ -18,7 +18,7 @@
 #include <curses.h>
 #include <term.h>
 #include <termios.h>
-#include <termcap.h>
+/*#include <termcap.h>*/
 #include <langinfo.h>
 #include <sys/types.h>
 #include <sys/stat.h>
